Unix/regress

```diff
@@ -228,7 +253,7 @@ cd omi-$version
 ## Create the prefix directory:
 ##
 
-prefix=/tmp/OMI932E75578CAB46E4A3EB87787B9EA40F
+prefix=/tmp/OMI932E75578CAB46E4A3EB87787B9EA40F-$USER
```

Comment:
If there's no reason for this name, then it's needlessly confusing. Just pick a reasonable name, like `/tmp/$USER-omi` or, if that conflicts already, then `/tmp/$USER-omi2` or whatever.